### ISSUE TITLE
Originality of code?

### ISSUE THREAD

**Issue by toomuchcodehelp** [Mr-Bamboo-Forest/Notes#1]
this code looks like it has been taken from other sites - which isn't a problem, it's just nice to leave a comment to help other know the original source

**Comment by Mr-Bamboo-Forest** [comment]
Yes I agree that, however some modifications to the code have been made and i can not find the originality of the code. Many thanks if you can leave a site for me to include.
